build.gradle.kts

```diff
@@ -84,6 +86,19 @@ sonarqube {
 	}
 }
 
+moditect {
+	addMainModuleInfo {
+		version = project.version
+		overwriteExistingFiles.set(true)
+		jdepsExtraArgs.set(listOf("-q", "--multi-release", "9"))
```

Comment:
Is this "jdeps"-related configuration still needed?

Reply:
> Is this "jdeps"-related configuration still needed?

@siordache Are they? I just copied them from your example 🤷‍♂️

Reply:
No, with 1.0.0-rc3 they are no longer needed.